ipatests/test_integration/test_cert.py

```diff
@@ -257,6 +258,16 @@ def test_getcert_list_profile_using_subca(self, test_subca_certs):
             raise AssertionError("certmonger request is "
                                  "in state {}". format(status))
 
+    def test_getcert_notafter_output(self):
+        """Test that currrent certmonger includes NotBefore in output"""
+        result = self.master.run_command(["certmonger", "-v"]).stderr_text
+        if parse_version(result.split['1']) < parse_version('0.79.14'):
```

Comment:
Used string instead of number here. `result.split[1]`

Reply:
It's quite amazing how wrong I got that :-( Thanks.